lambda/jvm/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -49,11 +49,8 @@ private[lambda] abstract class IOLambdaPlatform[Event, Result]
             .lastOrError
           context <- IO(Context.fromJava[IO](context))
           _ <- OptionT(lambda(event, context)).foreachF { result =>
-            IO {
-              val json = result.asJson
-              val writer = new OutputStreamWriter(output)
-              Printer.noSpaces.unsafePrintToAppendable(json, writer)
-              writer.close()
+            Resource.fromAutoCloseable(IO(new OutputStreamWriter(output))).use { writer =>
+              IO(Printer.noSpaces.unsafePrintToAppendable(result.asJson, writer))
```

Comment:
Hmm, this should probably be blocking 🤔 

Also I tried writing a test, but it would require injecting an error into this line somehow. In practice it would probably involve the unsafe printing somehow.